### PR TITLE
layer-2-scaling-survey-edits

### PR DESCRIPTION
**Pull request by anselld**
Start of editing process. This will be presented in a series of change reviews, starting with formatting only, and then proceeding with technical editing changes.

### CODE REVIEW

**Comment by anselld** [comment]
Hi @hansieodendaal, I am starting the editing process on https://tlu.tarilabs.com/layer2scaling/layer2scaling-landscape/layer2scaling-survey.html.

Will let you know when I am ready for your input and/or review comments.

Thanks,
Denise.

**Comment by hansieodendaal** [comment]
Reviewed and fixed references, some layout changes.

**Comment by anselld** [comment]
Hi Hansie, I've edited Layer 2 Scaling Survey (Part 1). Please let me know what you think.

**Comment by hansieodendaal** [comment]
Hi @anselld, thanks, will review.

_Note: Please do not close the pull requests, as the content must still be reviewed and merged in. I suspect you selected the 'Close and Comment' button when you left your latest comment._

image

**Comment by anselld** [comment]
Thanks Hansie - noted.

**Comment by anselld** [comment]
Thanks Hansie. Please see my recommendations.
Unsure what you mean by "incorrect "et al." form.

File: src/layer2scaling/layer2scaling-landscape/layer2scaling-survey.md [hunk 1]
```diff
@@ -392,7 +392,7 @@ Golem cites TrueBit as a verification mechanism for its forthcoming outsourced c
 
 #### Weaknesses
 
-TrueBit is yet to be fully tested. 
+At the time of writing (July 2018) TrueBit was not fully tested yet.
```
Comment (anselld) [inline src/layer2scaling/layer2scaling-landscape/layer2scaling-survey.md]: TrueBit had not yet been fully tested.

**Comment by hansieodendaal** [comment]
Perfect @anselld , all good! Merging this in to master.

**Pull request merged**